### ISSUE TITLE
Example code fails (python 3.4.3)

### ISSUE THREAD

**Issue by alfredr** [josiahcarlson/rom#63]
Any idea what's going on here? Not sure what the recommended fix is for unique Strings on python 3.4...

```
In [10]: %paste
class User(rom.Model):
    email = rom.String(required=True, unique=True, suffix=True)
    salt = rom.String()
    hash = rom.String()
    created_at = rom.Float(default=time.time)

## -- End pasted text --
---------------------------------------------------------------------------
ColumnError                               Traceback (most recent call last)
<ipython-input-10-19af8c163095> in <module>()
----> 1 class User(rom.Model):
      2     email = rom.String(required=True, unique=True, suffix=True)
      3     salt = rom.String()
      4     hash = rom.String()
      5     created_at = rom.Float(default=time.time)

<ipython-input-10-19af8c163095> in User()
      1 class User(rom.Model):
----> 2     email = rom.String(required=True, unique=True, suffix=True)
      3     salt = rom.String()
      4     hash = rom.String()
      5     created_at = rom.Float(default=time.time)

/Users/arthur/Documents/venvs/the-thing/lib/python3.4/site-packages/rom/columns.py in __init__(self, required, default, unique, index, keygen, prefix, suffix)
    210         if unique:
    211             if not (is_string or is_integer):
--> 212                 raise ColumnError("Unique columns can only be strings or integers")
    213
    214         numeric = True

ColumnError: Unique columns can only be strings or integers
```

**Comment by josiahcarlson** [comment]
There's a bug in rom, the fix is on my side. I'll try to get you a fix by Monday afternoon.

**Comment by josiahcarlson** [comment]
So I released 0.31.1, but there are a couple other bugs that I didn't catch, so I also just released 0.31.2 to fix the remaining bugs. You should use 0.31.2 .

You will also need to provide a "keygen" argument for your suffix index. For email addresses, the built-in `rom.FULL_TEXT` should work unless your emails have spaces in them (the email RFC allows quoted spaces), then this one is better: `lambda x:set([x.decode('latin-1')])` .
